docs/environment-setup.md

```diff
@@ -168,6 +168,14 @@ _Note: If experiencing an error such as `Error starting userland proxy: listen t
 
 To run Elasticsearch natively, follow the instructions for your OS [here](https://www.elastic.co/guide/en/elastic-stack-get-started/7.6/get-started-elastic-stack.html#install-elasticsearch)
 
-### Finally
+### Finally:
 
-Run `npm start` and open `localhost:3000`
+Telescope requires a running back-end and front-end to start. The following steps assume you are running the back-end locally. For more information on running the front-end with our staging/production as the back-end, please refer to our [Front-End Document](gatsbyjs.md).
+
+Start the back-end for Telescope:
+Run `npm start`
+
+Build the front-end for Telescope:
```

Comment:
Add a comment here about starting a second terminal so people don't try to do this in the first one.